### DISCUSSION TITLE
Can't concatenate CameraCapture and FileCapture

### DISCUSSION THREAD

**Discussion by ZacariasBarreto** [bonsai-rx/bonsai#1815] [Q&A] [answered]
Hi,

When I use a script joining CameraCapture and FileCapture using a WithLatestFrom node, and then a Concat node, I get the following Runtime error "Input arrays must have the same shape except in the dimension corresponding to axis". What does this error mean in terms of joining videos, and what could I do to get rid of this error?

Thanks in advance,
zbx

**Comment by bruno-f-cruz** [comment, marked as answer]
Your frames should have same size along the dimension you are merging (ie. the one you do not have specified in the operator)

**Comment by ZacariasBarreto** [comment]
The dimension I am merging changes with the number of the axis i choose in the Concat node, right?
And is there a node to crop the video coming from the camera to make it have the same size along the dimensions I am merging?
Thank you so much,

**Reply by bruno-f-cruz** [reply]
uhm not sure I follow. You can only concatenate along a single axis at a time.
You can crop the image using the `Crop` node.

**Reply by ZacariasBarreto** [reply]
I meant that in the Concat node, you can choose a number that changes the axis the videos are concatenated along.
Anyway, it's solved now
Thank you
